R/nm-join-bayes.R

```diff
@@ -418,6 +418,8 @@ sample_exts <- function(mod, n_post, min_batch_size) {
 
 sim_epred <- function(mod_mrgsolve, exts, data, join_col, y_col, pbar) {
   theta_cols <- grep("^THETA[0-9]+$", colnames(exts[[1]]))
+  # Drop post-hoc ETAs so that ETAs are randomly generated within model.
+  data <- data[grep("^ETA[0-9]+$", colnames(data), invert = TRUE)]
```

Comment:
@timwaterhouse Is this a situation where we need to worry about ETAs over a certain length (10?) being truncated to `ET`?

Reply:
@kyleam yes, good point. It should match `ET10` etc as well